ebpf/accesslog.proto

```diff
@@ -327,6 +331,11 @@ enum AccessLogKernelReadSyscall {
     RecvMmsg = 5;
 }
 
+enum AccessLogProtocolType {
+    Unknown = 0;
+    HTTP = 1;
+}
```

Comment:
No TCP? Is this a L7 protocol?

Reply:
If the Unknown It should be TCP. Or should I update the Unknown to the TCP?